driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -112,6 +116,33 @@ public Host getQueriedHost() {
     }
```

Comment:
The javadocs for `getQueriedHost()` should be amended: `... This is a shortcut for {@code getTriedHosts().get(getTriedHosts().size() - 1)}.`